src/main/scala/com/raphtory/serialisers/Marshal.scala

```diff
@@ -1,5 +1,50 @@
 package com.raphtory.serialisers
 
+
+/**
+  * {s}`Marshal()`
+  *
+  * : support serialisation (`dump`) and deserialisation (`load`) for `ByteArray` input and output streams
+  *
+  *
+  * ## Methods
+  *
+  *    {s}`dump[A](o: A)(implicit t: ClassTag[A]): Array[Byte]`
+  *      : Serialise to ByteArray
+  *
+  *        {s}`t: ClassTag[A]`
+  *           : serialise object to Array[Byte]
```

Comment:
`t` is not the object being serialised, in fact `t` should almost never be given explicitly as it is a ClassTag that normally should be passed in automatically by the compiler